kubejs/server_scripts/create.js

```diff
@@ -2620,19 +2615,19 @@ function mechanicalCraftingRecipes(event) {
 function pressingRecipes(event) {
     // [Input string, Output string]
     [
-        ["techreborn:silver_ingot", "techreborn:silver_plate"],
-        ["techreborn:tin_ingot", "techreborn:tin_plate"],
-        ["techreborn:lead_ingot", "techreborn:lead_plate"],
-        ["techreborn:electrum_ingot", "techreborn:electrum_plate"],
-        ["createastral:bronze_ingot", "createastral:bronze_sheet"],
+        //["techreborn:silver_ingot", "techreborn:silver_plate"],
+        //["techreborn:tin_ingot", "techreborn:tin_plate"],
+        //["techreborn:lead_ingot", "techreborn:lead_plate"],
+        //["techreborn:electrum_ingot", "techreborn:electrum_plate"],
+        //["createastral:bronze_ingot", "createastral:bronze_sheet"], // Moved to plates.js
         ["minecraft:lapis_block", "create:lapis_sheet"],
         ["createastral:pure_star_shard", "minecraft:nether_star"],
     ].forEach((recipe) => {
         event.recipes.createPressing(recipe[1], recipe[0]);
     });
 }
 function farmersDelightIntegration(event) {
-    let knivesTag = Platform.isFabric ? "c:tools/knives" : "forge:tools/knives";
+    let knivesTag = "c:tools/knives"; // We are on Fabric, so why the check is here?
```

Comment:
This is a comment for a PR, not for the code. It should probably be removed from the code.

Reply:
Removed the comment